packages/toolkit/.changes/next-release/Feature-eb088612-3f07-4410-86f2-6e91131d3ffe.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "CloudWatch Logs: Added support for Live Tailing LogGroups."
```

Comment:
are there command names we can mention here? or other entrypoints ("right-click Foo and select Bar...")

feel free to add multiple changelogs too

Reply:
I can add more detail!